tests/test_generic.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+from pyro.generic import infer, pyro_backend, seed
+from pyro.generic.testing import MODELS
+
+
+pytestmark = pytest.mark.stage('unit')
+
+
+@pytest.mark.parametrize('model', MODELS)
+@pytest.mark.parametrize('backend', ['pyro'])
```

Comment:
I think we can add a check if `numpyro` is available here. It is helpful to test locally.

Reply:
The idea is that each backend should be able to import these set of models and run tests as they see fit, so the tests for the numpy backend can be in numpyro. Of course, right now the issue is that we will need to install pyro first to be able to use this module - I have a suggestion in the design doc for this.